crates/db_views/src/structs.rs

```diff
@@ -106,6 +106,7 @@ pub struct RegistrationApplicationView {
 pub struct SiteView {
   pub site: Site,
   pub counts: SiteAggregates,
+  pub languages: Vec<Language>
```

Comment:
counts are allowed here because they come back with the join, but not languages, move those up to GetSiteResponse.